src/actions/index.js

```diff
@@ -560,180 +567,178 @@ function updateOutput(
              * if it's not visible, then ignore the rest of the updates
              * to the store
              */
-            if (!has(outputComponentId, getState().paths)) {
-                return;
-            }
 
-            // and update the props of the component
-            const observerUpdatePayload = {
-                itempath: getState().paths[outputComponentId],
-                // new prop from the server
-                props: data.response.props,
-                source: 'response',
-            };
-            dispatch(updateProps(observerUpdatePayload));
+            const {paths} = getState();
 
-            dispatch(
-                notifyObservers({
-                    id: outputComponentId,
-                    props: data.response.props,
-                })
-            );
+            Object.keys(data.response).map(k => [k, data.response[k]]).forEach(([k, props]) => {
```

Comment:
Iterate over each key of the new response payload, they are now the output prop name.